libs/template/file.go

```diff
@@ -68,13 +76,22 @@ func (f *copyFile) PersistToDisk() error {
 		return err
 	}
 	defer srcFile.Close()
-	dstFile, err := os.OpenFile(path, os.O_CREATE|os.O_EXCL|os.O_WRONLY, f.perm)
-	if err != nil {
+
+	if runsOnDatabricks() && f.isNotebook() {
+		content, err := io.ReadAll(srcFile)
+		if err != nil {
+			return err
+		}
+		return writeNotebook(path, content)
+	} else {
```

Comment:
style: We can get rid of the else block since the if block above returns.

Reply:
made the whole logic easier by creating `writeFile`